README.md

```diff
@@ -73,56 +82,75 @@ eclipse
 
 ![step8](./images/8-SelectWizard.png)
 
-#### Step 4.9: Tell it which projects to import (cli_test is the only one you need)
+#### Step 4.9: Select projects to import
+`cli_test` is the only one you need.
 
 ![step9](./images/9-ImportProjects.png)
 
-#### Step 4.10: What you should see after slected 'Finish'
+#### Step 4.10: Select 'Finish'
+You should see something like this after selecting 'Finish'.
 
 ![step10](./images/10-ProjectsAreSetup.png)
 
-### Step 5: Import launch configuration.
+### Step 5: Configure the Toolchain
+This step configures Eclipse so that it finds the CORE-V toolchain that was downloaded and installed in **Step 1**.
+
+In the Eclipse Project Explorer select the `cli_test` Project, right click and select `Properties` (typically appears at the bottom of the pop-up menu).
+Select `C/C++ Build / Settings` to bring up the window below:
+
+![Step 5.1](./images/5p1-Properties_for_cli_test.png)
+
+In this example, the `Toolchain path` was changed by selecting the `workspace` preference page.
+This brings up the "Workspace RISC-V Toolchains Paths" preferences selection window.
+Update the Toolchain folder to point to your install path.
+
+![Step 5.2](./images/5p2-WorkspaceRISC-VToolchainsPaths.png)
+
+### Step 6: Compile cli_test
+Assuming everything above has completed this step is simplicity itself.
+With `cli_test` selected in the Project Explorer pane, click on Build (the hammer icon):
+
+![Step6.0](./images/BuildCliTest.png)
+
```

Comment:
the Build was fine with 0 errors, but I had the same 1049 warnings as shown in the image

Reply:
If we want to get those warning elimnated, we will need to push very hard.